forwarder/update_handlers/ca_update_handler.py

```diff
@@ -32,6 +34,20 @@ def __init__(
         sub = self._pv.subscribe(data_type="time")
         sub.add_callback(self._monitor_callback)
 
+        ctrl_sub = self._pv.subscribe(data_type="control")
+        ctrl_sub.add_callback(self._unit_callback)
+
+    def _unit_callback(self, sub, response: ReadNotifyResponse):
+        old_unit = self._current_unit
+        try:
+            self._current_unit = response.metadata.units.decode("utf-8")
+        except AttributeError:
+            return
```

Comment:
Would it be useful to add a self._logger.error("some message about attribute error") here?

Reply:
I do not think so. Checking actual PVs in use, there are A LOT (maybe even a majority) that does not have a unit set which will then result in an AttributeError. This change will thus result in a message that will be too common and thus useless.